root/user/ReportUser.js

```diff
@@ -154,13 +154,11 @@ const ReportUser = ({
             help={
               <p>
                 {exp.l(
-                  `If you don’t want our admins to contact you further
-                   regarding this report, you can uncheck the checkbox
-                   above.
+                  `If you don’t want our admins to contact you 
+                   regarding this report, uncheck the checkbox above.
                    <br />
-                   We recommend leaving it checked, so that you can be
-                   contacted if the report is resolved or the admins
-                   need more information.`,
+                   Allowing admins to contact you
+                   makes your report more likely to be resolved.`,
```

Comment:
The previous version was more clear about our recommendation to not uncheck it.

The first sentence of the new version can be taken as an advice to uncheck it, the user might not even go to the second sentence.

If it absolutely needs to be simplified, I would suggest to swap sentences (and add a linking word such as “But” in between). This way it would start with what this is useful for, and then add that it is still possible to opt-out if wanted.

Reply:
We talked on IRC and realized there was a miscommunication between me, @Aerozol and @yvanzo about this bit, so dropped this commit for now. We can revisit this part later.